prow/cmd/deck/static/pr-script.js

```diff
@@ -146,9 +146,17 @@ function createSearchCard() {
     input.rows = 1;
     input.spellcheck = false;
     input.addEventListener("keydown", (event) => {
-        const el = event.target;
-        el.style.height  = "auto";
-        el.style.height = event.target.scrollHeight + "px";
+        if (event.keyCode === 13) {
+            event.preventDefault();
+
+            const query = getPRQuery(input.value);
```

Comment:
we should probably refactor this out into a method for submitting the query

Reply:
done